src/index.html

```diff
@@ -1,13 +1,51 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.css">
+  </head>
+  <body>
+    <article class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="product image"
+        class="card__img"
+      >
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__attribure">
+        Product code: 195434
+      </p>
```

Comment:
</p>

      <div class="card__raiting">
        <div class="stars">
          <div class="stars__star stars__star--active"></div>
          <div class="stars__star stars__star--active"></div>
          <div class="stars__star stars__star--active"></div>
          <div class="stars__star stars__star--active"></div>
          <div class="stars__star"></div>
        </div>

        <div class="card__review">Reviews: 5</div>

Add empty lines between **_multiline_** sibling blocks of HTML.

Reply:
Mentor review in layout stars component I had to add spaces only between blocks of code. You can see it here https://github.com/mate-academy/layout_stars/pull/1840#discussion_r984492377